src/mono/mono/metadata/loader-internals.h

```diff
@@ -62,6 +72,54 @@ struct _MonoAssemblyLoadContext {
 };
 #endif /* ENABLE_NETCORE */
 
+struct _MonoMemoryManager {
+	MonoDomain *domain;
+	// Whether the MemoryManager can be unloaded on netcore; should only be set at creation
+	gboolean collectible;
+	// Whether this is a singleton or generic MemoryManager
+	gboolean is_generic;
+	// Whether the MemoryManager is in the process of being freed
+	gboolean freeing;
+
+	// Entries moved over from the domain:
```

Comment:
This is a historical comment, its not really needed.

Reply:
Will remove.